hedera-node/test-clients/src/main/resources/contract/contracts/CircularTransfers/CircularTransfers.json

```diff
@@ -1,43 +1 @@
-[
-  {
-    "inputs": [],
-    "payable": true,
-    "stateMutability": "payable",
-    "type": "constructor"
-  },
-  {
-    "constant": false,
-    "inputs": [
-      {
-        "internalType": "uint32",
-        "name": "keepAmountDivisor",
-        "type": "uint32"
-      },
-      {
-        "internalType": "uint256",
-        "name": "stopBalance",
-        "type": "uint256"
-      }
-    ],
-    "name": "receiveAndSend",
-    "outputs": [],
-    "payable": true,
-    "stateMutability": "payable",
-    "type": "function"
-  },
-  {
-    "constant": false,
-    "inputs": [
-      {
-        "internalType": "uint64[]",
-        "name": "accounts",
-        "type": "uint64[]"
-      }
-    ],
-    "name": "setNodes",
-    "outputs": [],
-    "payable": true,
-    "stateMutability": "payable",
-    "type": "function"
-  }
-]
\ No newline at end of file
+[{"inputs":[],"stateMutability":"payable","type":"constructor"},{"inputs":[{"internalType":"uint32","name":"keepAmountDivisor","type":"uint32"},{"internalType":"uint256","name":"stopBalance","type":"uint256"}],"name":"receiveAndSend","outputs":[],"stateMutability":"payable","type":"function"},{"inputs":[{"internalType":"uint160[]","name":"accounts","type":"uint160[]"}],"name":"setNodes","outputs":[],"stateMutability":"payable","type":"function"}]
```

Comment:
This is harder to read imo

Reply:
I can format the json but largely these files generally do not get read, IMO.